app/controllers/api/v1/requested_packages_controller.rb

```diff
@@ -32,12 +33,19 @@ def index
 
       api :POST, '/v1/requested_packages', "Create a cart item"
       def create
-        if params[:quantity].blank?
-          @requested_package.quantity = PackagesInventory::Computer.available_quantity_of(@requested_package.package)
-        end
```

Comment:
Can you add a test case to make sure the api handles missing params correctly ? : )

Reply:
sure